modules/virt-listing-vmis-cli.adoc

```diff
@@ -0,0 +1,17 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+
+[id="virt-listing-vmis-cli_{context}"]
+= Listing all virtual machine instances using the CLI
+
+You can list both virtual machines (VMs) and standalone virtual machine instances (VMIs) in your cluster by using the `oc` command-line interface (CLI).
+
+.Procedure
+
+. List all VMs and standalone VMIs by running the following command:
+
+----
+$ oc get vmis
```

Comment:
IMO, it no need to append `s` to `vmi`, just use `oc get vmi` is fine.
ref: https://docs.openshift.com/container-platform/4.2/cnv/cnv_users_guide/cnv-controlling-vm-states.html

Reply:
Making the correction to the command.